src/screens/Search.jsx

```diff
@@ -1,49 +1,73 @@
-import { Text, VStack, ScrollView, Button, HStack, Box } from 'native-base';
-import { useState } from 'react';
+import { Center, Heading } from 'native-base';
+import { useEffect, useState } from 'react';
 import { Keyboard, TouchableWithoutFeedback } from 'react-native';
+import Header from '../components/Header';
+import LoadingSkeleton from '../components/LoadingSkeleton';
+import PaginationList from '../components/PaginationList';
+import RepoCard from '../components/RepoCard';
 import SearchBar from '../components/SearchBar';
 import useSearch from '../hooks/useSearch';
+import MainLayout from '../layouts/MainLayout';
 
 export default function SearchScreen() {
+  const [page, setPage] = useState(1);
+  const [shrink, setShrink] = useState(false);
   const [list, setList] = useState([]);
   const getSearchResult = useSearch();
-  const handleSearch = (keyword) => {
-    getSearchResult(keyword).then(setList);
+  const [result, setResult] = useState('');
```

Comment:
Recommend-Request: 검색 키워드 값에 대한 state로 보이는데 아래와 같이 작성하는 건 어떨까요?
```suggestion
  const [keyword, setKeyword] = useState('');
```
아래 쪽에서`{result ? 'No Repository!' : 'Find Your Favorite Repository!'}` 와 같이 값이 체크되는데, keyword가 있음에도 검색 결과가 없을 때(list.length ===0) 'No Repository!' 가 나타나야 한다는 의미로  사용하신 것 같아서 `result` 보다는 `keyword`가 좀 더 와닿지 않을까 생각합니다!

Reply:
`handleSearch`에서 이미 `keyword`를 사용하고 있어서 동일한 이름을 피하기 위해 `result`로 했습니다.